TemplateApplication/TemplateApplicationStandard.swift

```diff
@@ -155,48 +164,44 @@ actor TemplateApplicationStandard: Standard,
                 logger.error("Could not store consent form.")
                 return
             }
-            
+
             let metadata = StorageMetadata()
             metadata.contentType = "application/pdf"
-            _ = try await userBucketReference.child("consent/\(dateString).pdf").putDataAsync(consentData, metadata: metadata)
+            _ = try await userBucketReference
+                .child("consent/\(dateString).pdf")
+                .putDataAsync(consentData, metadata: metadata) { @Sendable _ in }
         } catch {
             logger.error("Could not store consent form: \(error)")
         }
     }
 
-
-    func create(_ identifier: AdditionalRecordId, _ details: SignupDetails) async throws {
-        guard let accountStorage else {
-            preconditionFailure("Account Storage was requested although not enabled in current configuration.")
-        }
-        try await accountStorage.create(identifier, details)
-    }
-
-    func load(_ identifier: AdditionalRecordId, _ keys: [any AccountKey.Type]) async throws -> PartialAccountDetails {
-        guard let accountStorage else {
-            preconditionFailure("Account Storage was requested although not enabled in current configuration.")
+    private func setupTestAccount() async {
```

Comment:
Wondering if we should also move this into a dedicated Firebase/Firestore Module that handles this and the firestorm paths and more?